src/AggregateNugetPackages/Microsoft.Net.RoslynDiagnostics/NuGet/Microsoft.Net.RoslynDiagnostics.props

```diff
@@ -0,0 +1,62 @@
+<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ItemGroup Condition="'$(ProjectLanguage)' == 'CSharp'">
+    <Analyzer Include="$(NuGetPackageRoot)\Microsoft.ApiDesignGuidelines.Analyzers\$(RoslynDiagnosticsNugetPackageVersion)\analyzers\dotnet\cs\Analyzer.Utilities.dll" />
```

Comment:
As per the current design, we also need to specify dependencies of analyzer assemblies as analyzers in the project file. This is how VS live analysis knows to shadow copy the dependencies next to actual analyzer assemblies.